AEXS/aex-141.md

```diff
@@ -37,7 +37,8 @@ contract interface NFT =
         , metadata_type : metadata_type}
 
     datatype event 
-        = Transfer(address, address, int)
+        = Mint(address, int)
+        | Transfer(address, address, int)
```

Comment:
Burn to be added

Reply:
actually I think we don't have to add it here because `Burn` is a separate extension. that's why it's not included here.

Reply:
but I have to re-think the whole document structure in that regards I guess 🤔 will have a look at AEX-9 again. but maybe it's also a bit messed up there

Reply:
`Burn` event only relevant for `burnable` extension